clients/js/genapi.sh

```diff
@@ -1,31 +1,18 @@
 #!/usr/bin/env sh
 
-# curl -s http://localhost:8000/openapi.json | jq > openapi.json
-curl -s http://localhost:8000/openapi.json | python -c "import sys, json; print(json.dumps(json.load(sys.stdin), indent=2))" > openapi.json
-
-if [[ "$OSTYPE" == "darwin"* ]]; then
-  # macOS
-  sed -i '' 's/"schema": {}/"schema": {"type": "object"}/g' openapi.json
-  sed -i '' 's/"items": {}/"items": { "type": "object" }/g' openapi.json
-  sed -i '' -e 's/"title": "Collection Name"/"title": "Collection Name","type": "string"/g' openapi.json
-else
-  # Linux
-  sed -i 's/"schema": {}/"schema": {"type": "object"}/g' openapi.json
-  sed -i 's/"items": {}/"items": { "type": "object" }/g' openapi.json
-  sed -i -e 's/"title": "Collection Name"/"title": "Collection Name","type": "string"/g' openapi.json
-fi
+curl -s https://api.trychroma.com:8000/openapi.json | jq > openapi.json
```

Comment:
Should this be off localhost? Otherwise how can you update it before releasing to this url? Also we should not, as a practice couple this repo to chroma cloud. Chroma cloud does not exist to the OSS repo.

Reply:
I think its ok to assume tilt is running

Reply:
Thanks - will update!